support-frontend/assets/pages/contributions-landing/contributionsLandingInit.ts

```diff
@@ -49,6 +50,8 @@ import {
 } from './contributionsLandingActions';
 import type { State } from './contributionsLandingReducer';
 
+type Action = ContribAction | CommonAction;
+
```

Comment:
Is there a better way of doing this? I don't think I've actually had to fix a file before that dispatched actions for reducers defined in multiple files. Here I've had to combine the common & contributionsLanding reducer actions into a single type.

Reply:
Type issues around `Dispatch` and `Action` are one of the things Redux Toolkit will help us with- it essentially enables you to [infer types for these things from your base store](https://redux-toolkit.js.org/usage/usage-with-typescript). I agree that this is pretty clunky, but I think it's clear and functional enough that it's probably not worth spending more time on if we're going to be adopting Toolkit in the near future.

Reply:
Oooo just read through the bit about getting the types for `Dispatch` and `Action` and that looks great! Very excited that we're working on adopting this